catalog/resource_sql_table.go

```diff
@@ -431,6 +467,24 @@ func columnChangesCustomizeDiff(d *schema.ResourceDiff) error {
 					return fmt.Errorf("changing the 'type' of an existing column is not supported")
 				}
 			}
+		} else {
+			oldColsNameToMap := make(map[string]map[string]interface{})
+			newColsNameToMap := make(map[string]map[string]interface{})
```

Comment:
At least for the `newColsNameToMap`, can we use `DiffToStructPointer`? I know our infra doesn't support getting the old version yet.